ADAL/src/ADEnrollmentGateway.m

```diff
@@ -109,6 +117,29 @@ + (NSString *)enrollmentIdForUniqueAccountId:(NSString *)uniqueAccountId
     }];
 }
 
++ (NSString *)enrollmentIdIfAvailable
+{
+    // this will just return the first enrollment ID
+    return [ADEnrollmentGateway getEnrollmentIDForIdentifier:^BOOL(NSDictionary * __unused dic) {
+        return true;
+    }];
+}
+
++ (NSString*)enrollmentIDForHomeUserID:(NSString*) homeUserID userID:(NSString*) userID
+{
+    NSString* enrollmentID;
+    enrollmentID = homeUserID ? [ADEnrollmentGateway enrollmentIdForUserId:homeUserID] : nil;
+    if (enrollmentID)
```

Comment:
i guess we should use `[ADEnrollmentGateway enrollmentIdForUniqueAccountId:homeUserId];` for `homeUserId`. 
`UniqueAccountId` and `homeUserId` are actually referring to the same thing. Sorry that it causes confusion to you. I will talk to Olga see if we could rename `homeUserId` in cache item to `UniqueAccountId`:)

Reply:
Just talk to Olga, we are going to to rename both `uniqueAccountId` and `homeUserId` to `homeAccountId` :)